core/session.go

```diff
@@ -140,7 +142,7 @@ func (s *Session) Initialize(v *viper.Viper, scanType string) {
 	s.BindAddress = v.GetString("bind-address")
 	s.BindPort = v.GetInt("bind-port")
 	s.CommitDepth = setCommitDepth(v.GetInt("commit-depth"))
-	//s.CSVOutput = v.GetBool("csv")
```

Comment:
@funkwhatyouheard This should stay **CSVOutput** and **JSONOutput**. The reason being, *s.JSON* and *s.CSV* could mean several things as we scale. Is that an input file? Is it an output file? Is it some sort of data type used with protobuffers? The more specific we can be, just like with declaring the type, the easier bugs will either not occur, or they will be floated up.

This also makes the code easier to grok for people new to the tool as it is already becoming fairly in-depth with lots of little gotcha's. Low level git stuff is hard to wrap your head around sometimes IMHO, the other bits should be made as easy as possible to encourage further development.

Reply:
You thinking change the struct and the cli flag or just the struct?